docs/integrations/sources/sharetribe.md

```diff
@@ -0,0 +1,32 @@
+# Sharetribe
+Sharetribe source connector for using the sharetribe API. 
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `client_id` | `string` | Client ID.  |  |
+| `client_secret` | `string` | Client secret.  |  |
+| `client_refresh_token` | `string` | Refresh token.  |  |
+| `oauth_access_token` | `string` | Access token. The current access token. This field might be overridden by the connector based on the token refresh endpoint response. |  |
+| `oauth_token_expiry_date` | `string` | Token expiry date. The date the current access token expires in. This field might be overridden by the connector based on the token refresh endpoint response. |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| users | id | No pagination | ✅ |  ❌  |
```

Comment:
Please confirm Pagination doesn't work to all endpoints: https://www.sharetribe.com/api-reference/#pagination

Reply:
Added pagination support for endpoints